Filters/exceptions.txt

```diff
@@ -1,3 +1,5 @@
+! https://github.com/AdguardTeam/AdguardFilters/issues/166702
+@@||omns.americanexpress.com^|
```

Comment:
Is it blocked by DNS?
I cannot find any rule which would block it, but maybe I missed something.

Reply:
Not finding it in DNS filter

Reply:
Yeah, I also don't see it.

Reply:
hmm very strange, can't find it either. Shouldn't it be blocked by cname?

Reply:
It probably should be blocked by `||omtrdc.net^`, but it's excluded:
https://github.com/AdguardTeam/AdGuardSDNSFilter/blob/40409378d690010ecd63d2dd13e47398f6373184/Filters/exclusions.txt#L865

Reply:
Maybe this will fix the problem
https://github.com/AdguardTeam/AdguardFilters/commit/15aaf0f3bb519b16dc3bb8519285256547a3483b
https://filters.adtidy.org/ios/filters/3_optimized.txt

Reply:
At the moment it is not blocked
![image](https://github.com/AdguardTeam/AdGuardSDNSFilter/assets/8361299/2139ddb2-b98f-4486-9fe1-52f7f778f9a2)